fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsSchedularInterestPoster.java

```diff
@@ -206,7 +206,8 @@ private void batchUpdate(final List<SavingsAccountData> savingsAccountDataList)
                             savingsAccountTransactionData.getBalanceNumberOfDays(), savingsAccountTransactionData.getRunningBalance(),
                             savingsAccountTransactionData.getCumulativeBalance(), currentDate, Integer.valueOf(1),
                             savingsAccountTransactionData.isManualTransaction(), savingsAccountTransactionData.getRefNo(),
-                            savingsAccountTransactionData.isReversalTransaction(), savingsAccountTransactionData.getOverdraftAmount(), });
+                            savingsAccountTransactionData.isReversalTransaction(), savingsAccountTransactionData.getOverdraftAmount(),
+                            savingsAccountTransactionData.getTransactionDate() });
```

Comment:
this looks wrong to me... submitted on date != transaction date

Reply:
@adamsaghy I think this should be DateUtils.getBusinessLocalDate() instead of the transaction date. Please correct me if i am wrong

Reply:
I believe so